tools/uw-frame-static/build.js

```diff
@@ -12,9 +12,12 @@ var exec_handler = function(error, stdout, stderr) {
 
 var mkdirp = require('mkdirp');
 var copy = require('recursive-copy');
+var copyOptions = {
+  overwrite: true
+}
 
-mkdirp('uw-frame-static/target/css/themes/', function (err) {
-  if (err) throw error;
+mkdirp('uw-frame-static/target/css/themes/', function (error) {
+  if (error) throw error;
```

Comment:
Got it, you are expecting Google code style's [`new-cap`](http://eslint.org/docs/rules/new-cap) lint rule to catch this.